src/main/webapp/i18n/de/metis.json

```
@@ -90,7 +90,9 @@
                 "TECH_SUPPORT": "Technische Hilfe",
                 "ORGANIZATION": "Organisation",
                 "RANDOM": "Sonstiges",
-                "ANNOUNCEMENT": "Ankündigung"
+                "ANNOUNCEMENT": "Ankündigung",
+                "allPublicMessages": "Alle öffentlichen Nachrichten",
+                "searchResults": "Suchergebnisse für {{ search }}"
```

Since the English translation has the quotation marks, do we want them here as well?

```suggestion
                "searchResults": "Suchergebnisse für '{{ search }}'"
```